source/includes/administration/_discounts.md

```diff
@@ -64,9 +64,67 @@ Attributes | &nbsp;
 `status`<br/>*enum* | The status of the discount. Possible values are: UPCOMING, CURRENT, ENDED.
 
+<!-------------------- GET DISCOUNT -------------------->
+### Get discount
 
-<!-------------------- CREATE DISCOUNTS -------------------->
-### Create discounts
+`GET /applied_pricings/:applied_pricing_id/discounts/:id`
+
+Retrieves a discount's details.
+
+```shell
+# Retrieve applied pricing list
+curl "https://cloudmc_endpoint/rest/applied_pricings/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts/08695c90-3e41-4be8-a3a3-df964620ba62" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "applyToNewCustomersOnly": true,
+      "discountedProducts": {},
+      "durationMonths": 22354,
+      "type": "PERCENTAGE",
+      "packageDiscount": 20.0,
+      "discountScope": "ALL_PRODUCTS",
+      "isDeactivated": true,
+      "discountedCategories": {},
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "appliedPricing": {
+        "id": "efd32752-c6f2-45cf-b494-cc6be8a45845"
+      },
+      "startDate": "2021-07-20T15:57:16.132Z",
+      "status": "CURRENT"
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the discount.
+`discountedProducts`<br/>*Map[UUID, BigDecimal]* | A mapping of the desired priced product IDs and discounts. All pricing products specified will have the discount value applied to them.
+`durationMonths`<br/>*integer* | Duration of the discount once it has been applied to a customer. If not provided the discount will last indefinitely, or until credit values are reached.
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+`packageDiscount`<br/>*BigDecimal* | The discount value that will be applied to all products within the applied pricing.
+`discountScope`<br/>*enum* | The scope of the discount. It could be either "ALL_PRODUCTS", "CATEGORIES" or "PRODUCTS".
+`isDeactivated`<br/>*boolean* | Whether or not the discount is deactivated. Defaults to false.
+`discountedCategories`<br/>*Map[UUID, BigDecimal]* | A mapping between category IDs and discount values. All pricing products within a category will have the discount value applied to them.
+`name`<br/>*Map[String, String]* | The name translations of the discount.
+`appliedPricing`<br/>*Object* | The applied pricing being discounted.
+`appliedPricing.id`<br/>*UUID* | The UUID of the applied pricing.
+`applyToNewCustomersOnly`<br/>*boolean* | If true, the discount will only be applied to organizations created after the discount start date.
+`startDate`<br/>*date* | The start date of the discount.
+`cutoffDate`<br/>*date* | The date on which the discount will no longer be available to customers who have not already received it. If not provided, the discount will always be available after the start date.
```

Comment:
just noticed this, given the discussion on Friday we should probably change available -> offered to be consistent

Reply:
8a8df4722c9c5598dfe5b0432eedf20acd1ba951